wurst/_handles/primitives/Real.wurst

```diff
@@ -21,6 +21,21 @@ public function real.sign() returns int
 public function real.round() returns int
 	return this > 0 ? (this+.5).toInt() : (this-.5).toInt()
 
+/** Rounds the input real to the nearest 'scale' divisible */
```

Comment:
Because it properly divides by any `real`. E.g. (3.9).round(1,5) == (4.5)

Reply:
Yeah, one usually expects round/floor functions to expect ints, not reals.
In any case, I'm not sure if the functions in general are a good fit. If I were to search for such functionality, I wouldn't search for "round". When I see `.round(412)` I'm not immediately understanding what exactly the parameter means. And since it's a simple one-liner it's usually implemented in-place anyway.
Do you even make use of roundUp/Down with scale yourself? Do you have any examples of this in other libraries?

Also, divisible might be a term, but I haven't seen it used in this context. Something like "Returns the nearest multiple of 'scale' from the input number" strikes me with much more understanding.